api/v1beta1/common_types.go

```diff
@@ -77,8 +77,8 @@ type WatcherCommon struct {
 	TopologyRef *topologyv1.TopoRef `json:"topologyRef,omitempty"`
 }
 
-// WatcherTemplate defines the fields used in the top level CR
-type WatcherTemplate struct {
+// WatcherTemplate defines the fields used from the OpenStackControlPlane CR (does not include images)
+type WatcherSpecCore struct {
```

Comment:
+1 for the name change to be more aligned with other (e.g [nova](https://github.com/openstack-k8s-operators/nova-operator/blob/c26e486539947163daedba560dad57a87f7ab776/api/v1beta1/nova_types.go#L29) or [cinder](https://github.com/openstack-k8s-operators/cinder-operator/blob/main/api/v1beta1/cinder_types.go#L116) ). Is that a strict requirement from the openstack-operator? The function description needs to be updated to use the new name, however

Reply:
I'm not sure how strict is, but i think all the service operators are using it. I'll update the description.

Reply:
thanks for the change, I was just curious, one way or another it's good to be aligned, I think this is good to go now